test/test_support.ts

```diff
@@ -173,6 +173,19 @@ export class GoldenFileTest {
         .map(f => path.join(this.path, GoldenFileTest.tsPathToJs(f)));
   }
 
+  /**
+   * Find the absolute path to the tsickle root directory by reading the
+   * symlink bazel puts into bazel-bin back into the test_files directory
+   * and chopping off the test_files portion.
+   */
+  get workspaceRoot(): string {
```

Comment:
given that this has substantial logic, can fail, and also some performance impact (reads the file system), can you make it a regular method, to reflect that?